examples/teleop/src/main.ts

```diff
@@ -24,6 +24,15 @@ el("button").addEventListener("click", async () => {
 
     // get current device from url context
     const device = await Fleet.getCurrentDevice();
+    
+    // report connections and disconnections
+    window.device.on('connect', () => {
+        console.log('Connected!');
+      });
+
+      window.device.on('disconnect', () => {
```

Comment:
If you can, try to match the whitespace of the rest of the lines. Sometimes you need to use spaces instead of tabs.